benchmarks/defs/defs.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+message Payload {
```

Comment:
Rename defs to proto.

Name the proto file with what it is, e.g. ping.proto.

Reply:
Nice, let me do that.